### PR TITLE
Null pointer exception when invalid metadata from btcpay

### PR DESCRIPTION
**Pull request by RodriFS**

### CODE REVIEW

**Review by markettes — APPROVED** [review]
LGTM. Although I would say it would be nice to have a message saying that it was invalid metadata.

File: src/Pages/Withdrawals.razor [hunk 1]
```diff
@@ -586,23 +586,15 @@
...
+                    int atIndex = userName.IndexOf('@');
+                    return atIndex >= 0 ? userName.Substring(0, atIndex) : userName;
                 }
-            }
-            catch (JsonReaderException ex)
-            {
-                Console.WriteLine("Invalid JSON: " + ex.Message);
+                return userName;
             }
         }
```
Comment (AleksSoftware) [inline src/Pages/Withdrawals.razor:600]: I suggest improving the code to be more readable:

 return walletWithdrawalRequest.UserRequestor?.ToString() ?? string.Empty;

**Comment by AleksSoftware** [comment]
LGTM🚀

File: src/Pages/Withdrawals.razor (hunk 1 above)
Comment (Jossec101) [reply src/Pages/Withdrawals.razor:600]: Similar, defer to an issue.

**Pull request merged**